.github/workflows/publish-cli-bins.yaml

```
@@ -9,57 +9,84 @@
+          # Split os-arch into os and arch
+          os_arch="${{ matrix.os_arch }}"
+          IFS="/" read -r GOOS GOARCH <<< "$os_arch"
+          NAME=flightctl-$GOOS-$GOARCH
+          
+          # Export variables to GITHUB_ENV for subsequent steps
+          echo "GOOS=$GOOS" >> $GITHUB_ENV
+          echo "GOARCH=$GOARCH" >> $GITHUB_ENV
+          echo "NAME=$NAME" >> $GITHUB_ENV
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
```

actionlint (1.7.4)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)